scripts/modules/b2b-account/quotes.js

```diff
@@ -154,7 +155,8 @@ define([
                         pageSelector: 'selectb2bAccount',
                         optionalFilter: 'and',
                         filterKey: 'isActive',
-                        filterValue: 'true'
+                        filterValue: 'true',
+                        userId: userId
```

Comment:
We can't "hardcode" anything like this. This component needs to be generic and work for _any mozu type or any set of filters_. Find another solution that is generic.

Work with Bhushan if you have questions about how this component works. He wrote it.

Reply:
Done